src/backend/parser/ag_scanner.l

```diff
@@ -1099,15 +1097,6 @@ static bool is_low_surrogate(const pg_wchar c)
     return (c >= 0xDC00 && c <= 0xDFFF);
 }
 
-// http://www.unicode.org/faq/utf_bom.html#utf16-4
-static pg_wchar surrogate_pair_to_codepoint(const pg_wchar hi,
```

Comment:
This function is used in line 538. Is it okay to remove its definition?

Reply:
It looks to be included via a header now -

`#include "mb/pg_wchar.h"`

```
  1 F   f    surrogate_pair_to_codepoint  /home/jgemignani/POSTGRESQL/postgresql-13.9/src/include/mb/pg_wchar.h
               surrogate_pair_to_codepoint(pg_wchar first, pg_wchar second)
```